src/main/java/seedu/planner/logic/parser/ParserUtil.java

```diff
@@ -70,6 +72,23 @@ public static ModuleCode parseModuleCode(String moduleCode) throws ParseExceptio
         return new ModuleCode(trimmedName);
     }
 
+    /**
+     * Parses a {@code String moduleCodes} into a list of {@code ModuleCode}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if any of the given {@code ModuleCode} is invalid.
+     */
+    public static List<ModuleCode> parseModuleCodes(String moduleCode) throws ParseException {
+        requireNonNull(moduleCode);
+        String trimmedName = moduleCode.trim();
+        String[] moduleCodeStrings = trimmedName.split(" ");
```

Comment:
Bug: doesn't handle whitespace correctly
![CS2103T multi-module fail spaces](https://user-images.githubusercontent.com/51158450/79051760-98c76680-7c64-11ea-8636-ede40cb5e18e.png)

Reply:
Fixed, committing in abit.

Reply:
Seems to be fixed.